int32 id
---
int32 angle_offset
int32 angle_min
int32 angle_max
float32 voltage_min
float32 voltage_max
float32 temperature_limit
bool motor_mode
int32 motor_speed
bool torque_enabled
bool led_enabled
bool led_error_temp  
bool led_error_voltage 
bool led_error_locked  
float32 current_temperature
float32 input_voltage
float32 physical_angle
float32 commanded_angle
